tools/deployed/mainnet.SwappaRouterV1.addr.json

```diff
@@ -1 +1 @@
-{"address":"0xF35ed7156BABF2541E032B3bB8625210316e2832"}
\ No newline at end of file
+{"address":"0xf821B3c10c922C88D12FF5f91Cf2275e0768D7Dc"}
```

Comment:
this is a placeholder address because the ts component takes it as an arg in constructor. It's still yet to be deployed.